### PR TITLE
CEDS-939 Notification status

### PR DESCRIPTION
**Pull request by PatrykRudnicki**

### CODE REVIEW

File: app/models/Status.scala [hunk 1]
```diff
@@ -17,38 +17,107 @@
...
 
   implicit object StatusFormat extends Format[Status] {
     def reads(status: JsValue): JsResult[Status] = status match {
-      case JsString("Accepted")  => JsSuccess(Accepted)
-      case JsString("Arrived")   => JsSuccess(Arrived)
-      case JsString("Departed")  => JsSuccess(Departed)
-      case JsString("PreLodged") => JsSuccess(PreLodged)
-      case JsString("Query")     => JsSuccess(Query)
-      case _                     => JsError("Incorrect value")
+      case JsString("01")   => JsSuccess(Accepted)
```
Comment (raghu1978) [inline app/models/Status.scala]: why Jsstring?

File: app/models/Status.scala [hunk 2]
```diff
@@ -17,38 +17,107 @@
...
 
-case object Query extends Status
+case object Released extends Status
+
+case object Cleared extends Status
+
+case object Cancelled extends Status
+
+case object CustomsPositionGranted extends Status {
+  override def toString(): String = "Customs Position Granted"
```
Comment (raghu1978) [inline app/models/Status.scala]: move these into ENUM please

File: test/controllers/supplementary/ConfirmationPageControllerSpec.scala [hunk 3]
```diff
@@ -53,24 +56,47 @@ class ConfirmationPageControllerSpec extends CustomExportsBaseSpec {
       resultAsString must include(messages("supplementary.confirmation.explanation"))
     }
 
-    "display LRN from flash" in new Test {
```
Comment (raghu1978) [inline test/controllers/supplementary/ConfirmationPageControllerSpec.scala]: are we using flash? shouldn't use the scope on MDTP please use cache

Comment (PatrykRudnicki) [reply test/controllers/supplementary/ConfirmationPageControllerSpec.scala]: It's actually implemented in previous task, not related with actual one. @MaciejRewera check this

File: app/models/Status.scala (hunk 2 above)
Comment (PatrykRudnicki) [reply app/models/Status.scala]: Why?

File: app/models/Status.scala (hunk 1 above)
Comment (PatrykRudnicki) [reply app/models/Status.scala]: so we're sending this as Json, the values will be JsString, code inside will be the same like in notifications

File: test/controllers/supplementary/ConfirmationPageControllerSpec.scala (hunk 3 above)
Comment (MaciejRewera) [reply test/controllers/supplementary/ConfirmationPageControllerSpec.scala]: If we shouldn't, then I will change it to cache in separate PR.

File: app/models/Status.scala (hunk 2 above)
Comment (PatrykRudnicki) [reply app/models/Status.scala]: https://medium.com/@yuriigorbylov/scala-enumerations-hell-5bdba2c1216

If you can I can only move it to `java.util.Enum`, but I don't see any pros of it

**Pull request merged**